### ISSUE TITLE
[google_maps_flutter][Android] Tap interactions fail when using GMM10 renderer and screen reader

### ISSUE THREAD

**Issue by ChristianEdwardPadilla** [flutter/flutter#112662]
## Steps to Reproduce

EDIT: google_maps_flutter version in use: 2.1.4. I will verify that this repros on the latest release (2.2.1).
Edit2: I verified this still occurs on the latest version.

Use google_maps_flutter plugin on Android. When a map is displayed on screen, and TalkBack is enabled, try to tap on a Flutter widget on screen.

**Expected results:**
You should be able to tap on the widget to focus it.

**Actual results:**
Taps are ignored. You can focus the widgets by swiping to them, but you cannot directly tap on them.

Sample app Dart main:
```dart
import 'package:google_maps_flutter/google_maps_flutter.dart';
import 'package:flutter/material.dart';

void main() => runApp(MaterialApp(home: MyApp()));

class MyApp extends StatefulWidget {
  const MyApp({super.key});

  @override
  State<MyApp> createState() => _MyAppState();
}

class _MyAppState extends State<MyApp> {
  final Map<String, Marker> _markers = {};

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: const Text('GoogleMaps examples')),
      body: Stack(
        children: [
          map,
          button,
        ],
      ),
    );
  }

  Widget get map => GoogleMap(
        onMapCreated: _onMapCreated,
        initialCameraPosition: CameraPosition(
          target: LatLng(0, 0),
          zoom: 2,
        ),
        markers: _markers.values.toSet(),
      );

  Widget get button => FloatingActionButton(
        onPressed: () {
          print('print');
          showDialog<void>(
            context: context,
            builder: (BuildContext context) {
              return AlertDialog(
                title: const Text('Basic dialog title'),
                content: const Text('A dialog'),
              );
            },
          );
        },
      );

  Future<void> _onMapCreated(GoogleMapController controller) async {
    setState(() {
      _markers.clear();
      final marker = Marker(
        markerId: MarkerId('test'),
        position: LatLng(1, 1),
        infoWindow: InfoWindow(
          title: 'test1',
        ),
      );
      _markers['test'] = marker;
    });
  }
}
```

Android main:
```java
package com.google.android.flutter.samples.helloflutter;

import android.os.Bundle;
import com.google.android.gms.maps.MapsInitializer;
import com.google.android.gms.maps.MapsInitializer.Renderer;
import io.flutter.embedding.android.FlutterActivity;
import io.flutter.embedding.engine.FlutterEngine;

/**
 * Activity is one of the main concepts in Android. It is used for initializing and driving UI.
 *
 * <p>You don't typically need to interact with the activity in Flutter. The most common use case is
 * when initializing plugins. For more information on activities, please see Android documentation.
 */
public class MainActivity extends FlutterActivity {
  @Override
  public void configureFlutterEngine(FlutterEngine flutterEngine) {
    PluginRegistrant.registerWith(flutterEngine);
  }

  @Override
  public void onCreate(Bundle bundle) {
    super.onCreate(bundle);
    MapsInitializer.initialize(getApplicationContext(), Renderer.LATEST, null);
  }
}
```

Manifest:
```
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.google.android.flutter.samples.helloflutter"
    android:versionCode="1"
    android:versionName="1.0.0" >

  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="27" />

  <!-- The INTERNET permission is required for development. Specifically,
         flutter needs it to communicate with the running application
         to allow setting breakpoints, to provide hot reload, etc. -->
  <uses-permission android:name="android.permission.INTERNET"/>

  <application
        android:label="@string/app_name"
        android:icon="@mipmap/ic_launcher"
               android:usesCleartextTraffic="true"
        android:taskAffinity="" >
    <!-- Flutter apps have a single activity that handles all the links between Flutter and Android.
     -->
    <activity
        android:name=".MainActivity"
        android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
        android:exported="true"
        android:hardwareAccelerated="true"
        android:launchMode="singleTop"
        android:theme="@style/LaunchTheme"
        android:windowSoftInputMode="adjustResize">
      <intent-filter>
        <action android:name="android.intent.action.MAIN" />
        <category android:name="android.intent.category.LAUNCHER" />
      </intent-filter>
    </activity>
<!--     Use your API KEY HERE -->
    <meta-data
      android:name="com.google.android.geo.API_KEY"
      android:value=""/>
  </application>

    <!-- Flutter needs internet permission to communicate with the running application
         to allow setting breakpoints, to provide hot reload, etc.
    -->
  <uses-permission android:name="android.permission.INTERNET"/>
  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
</manifest>
```

https://user-images.githubusercontent.com/37954976/193113715-dbbe5ff2-a36f-4cbb-a1e3-c2a23f33e474.mp4

**Comment by huycozy** [comment]
Hi @ChristianEdwardPadilla 
Thanks for the report! This is a known issue being tracked at #101887 (Platform-views + Accessibility). Closing this issue as a duplicate, so please follow up on it for further updates. Thank you!
